System.IO.Abstractions/System.IO.Abstractions.csproj

```diff
@@ -23,6 +23,12 @@
         <SignAssembly>True</SignAssembly>
     </PropertyGroup>
 
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0'">
```

Comment:
Is this necessary? If yes you could just add the condition to the item group below.

Reply:
Grouped into the other condition.